src/dune_engine/build_system.mli

```diff
@@ -85,26 +87,27 @@ val set_vcs : Vcs.t list -> unit Memo.Build.t
 
 (** [prefix_rules t prefix ~f] Runs [f] and adds [prefix] as a dependency to all
     the rules generated by [f] *)
-val prefix_rules : unit Action_builder.t -> f:(unit -> 'a) -> 'a
+val prefix_rules :
```

Comment:
Looks like the comments for `prefix_rules` and `eval_pred` got out of date (e.g. there is no `t` anymore). It may be worth fixing this while you're looking at this code.

Reply:
Indeed, fixed.

Reply:
There is one more (`eval_pred`) to fix.

Reply:
Indeed, I updated that as well (and removed the `[`..`]` around the glob argument, which didn't seem useful).